tests/unit/test_llm.py

```diff
@@ -48,8 +56,9 @@ def test_llm_init_with_model_info(mock_get_model_info, default_config):
         'max_output_tokens': 2000,
     }
     llm = LLM(default_config)
-    assert llm.config.max_input_tokens == 8000
-    assert llm.config.max_output_tokens == 2000
+    # model info has precedence over config!
+    assert llm.config.max_input_tokens == 4096
+    assert llm.config.max_output_tokens == 4096
```

Comment:
This seems curious... If model_info has precedence over config, then the mock model info at line 54 should have won